README.md

```diff
@@ -74,6 +74,9 @@ As of 2019, Pillow development is
             <a href="https://pypi.org/project/Pillow/"><img
                 alt="Number of PyPI downloads"
                 src="https://img.shields.io/pypi/dm/pillow.svg"></a>
+            <a href="https://bestpractices.coreinfrastructure.org/projects/6331"><img
+                alt="CII Best Practices"
```

Comment:
```suggestion
                alt="openssf best practices"
```
I think the alt text should reflect what is otherwise seen?

Reply:
Good idea, looks like they've not updated the template at https://bestpractices.coreinfrastructure.org/en/projects/6331 since they [renamed](https://bestpractices.coreinfrastructure.org/en):

> This project was formerly known as the Core Infrastructure Initiative (CII) Best Practices badge. and was originally developed under the CII. It is now part of the [OpenSSF](https://openssf.org/) [Best Practices Working Group (WG)](https://github.com/ossf/wg-best-practices-os-developers). The OpenSSF is a foundation of the [Linux Foundation (LF)](https://www.linuxfoundation.org/). The project was formally renamed from "CII Best Practices badge" on 2021-12-24.

I'll send them feedback, and also update the alts to use proper caps.

Reply:
Sent them a PR: https://github.com/coreinfrastructure/best-practices-badge/pull/1863